engine/src/main/scala/cromwell/webservice/routes/wes/WesRouteSupport.scala

```diff
@@ -0,0 +1,98 @@
+package cromwell.webservice.routes.wes
+
+import akka.actor.ActorRef
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.pattern.{AskTimeoutException, ask}
+import akka.util.Timeout
+import cromwell.engine.instrumentation.HttpInstrumentation
+import cromwell.services.metadata.MetadataService.{GetStatus, MetadataServiceResponse, StatusLookupFailed, StatusLookupResponse}
+import cromwell.webservice.routes.CromwellApiService.{EnhancedThrowable, UnrecognizedWorkflowException, validateWorkflowId}
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+import WesResponseJsonSupport._
+import akka.http.scaladsl.model.{StatusCode, StatusCodes}
+import akka.http.scaladsl.server.directives.RouteDirectives.complete
+import WesRouteSupport._
+import cromwell.core.WorkflowId
+import cromwell.engine.workflow.WorkflowManagerActor.WorkflowNotFoundException
+import cromwell.server.CromwellShutdown
+import cromwell.webservice.routes.CromwellApiService
+
+trait WesRouteSupport extends HttpInstrumentation {
+  val serviceRegistryActor: ActorRef
+  val workflowManagerActor: ActorRef
+  val workflowStoreActor: ActorRef
+
+  implicit val ec: ExecutionContext
+  implicit val timeout: Timeout
+
+  /*
+    Defines routes intended to sit alongside the primary Cromwell REST endpoints. For instance, we'll now have:
+    GET /api/workflows/v1/ID/status
+    GET /ga4gh/wes/v1/runs/ID
+
+    POST /api/workflows/v1/ID/abort
+    POST /ga4gh/wes/v1/runs/ID/cancel
+
+    These routes are not currently going through the MetadataBuilderRegulator/MetadataBuilder. This is for two reasons:
+      - It'd require a fairly substantial refactor of the MetadataBuilderActor to be more general
+      - It's expected that for now the usage of these endpoints will not be extensive, so the protections of the regulator
+         should not be necessary
```

Comment:
um

Reply:
Yeah, this isn't going to be subject to change, and practically speaking isn't a problem as the only place we deeply care about the regulator won't hit these endpoints.

The two alternatives are to massively refactor the metadata builder & regulator actors to not be so opinionated about the JSON coming out (or better yet, to not fully build the JSON) **or** the WES endpoints take the built JSON, parse it, pull the bits they want out and create new JSON.

This topic is going to come to a head in some later endpoints so I decided to wait and see how big the problem really is when I get there instead of making a gnarly executive decision now.